packages/glimmer-runtime/lib/environment.ts

```diff
@@ -116,9 +116,15 @@ export class Scope {
   }
 }
 
+interface ModifierPair {
```

Comment:
Not sure if it's better to alloc an object to hold the pair or to create 2 arrays to hold each separately.